docs/docs/aleph_zero/sdk.md

```diff
@@ -185,56 +182,38 @@ await psp22.approve(account, invariant.contract.address.toString(), tokenXAmount
 await psp22.setContractAddress(poolKey.tokenY)
 await psp22.approve(account, invariant.contract.address.toString(), tokenYAmount)
 
-// open up position
+// create position
 const createPositionResult = await invariant.createPosition(
   account,
   poolKey,
-  lowerTick,
-  upperTick,
+  lowerTickIndex,
+  upperTickIndex,
   positionLiquidity,
   initSqrtPrice,
   0n
 )
-console.log(createPositionResult.hash) // print transaction hash
+console.log(createPositionResult.hash)
 ```
 
-### Perform swap
+:::tip Output
+7999999999880n 8000000000000n <br/>
+0x652108bb36032bc386fec2eef3f483f29970db7bdbdc9a1a340e279abd626ee2
+:::
+
+### Swap tokens
 
 :::info How to calculate input amount
 In order to calculate input amount, we have to multiply actual token amount you want to swap times 10 to the power of decimal.
 Let's say some token has decimal of 12 and we want to swap 6 actual tokens. Here is how we can calculate input amount:
 6 \* 10^12 = 6000000000000.
 :::
 
-```typescript
-interface Position {
-  poolKey: PoolKey
-  liquidity: Liquidity
-  lowerTickIndex: bigint
-  upperTickIndex: bigint
-  feeGrowthInsideX: FeeGrowth
-  feeGrowthInsideY: FeeGrowth
-  lastBlockNumber: bigint
-  tokensOwedX: TokenAmount
-  tokensOwedY: TokenAmount
-}
-
-interface Tick {
-  index: bigint
-  sign: boolean
-  liquidityChange: Liquidity
-  liquidityGross: Liquidity
-  sqrtPrice: SqrtPrice
-  feeGrowthOutsideX: FeeGrowth
-  feeGrowthOutsideY: FeeGrowth
-  secondsOutside: bigint
-}
-```
+Peforming a swap requires: specifying the amount of tokens to be swapped, approving the transfer of the token, estimating the result of the swap, determining the allowed slippage, calculating the square root price limit based on slippage, and finally, executing the swap. Here's a breakdown of the process:
```

Comment:
"estimating the result of the swap"  - here, it is crucial to specify exactly what kind of estimation is required to pass. It's even important to note that the swap tolerance is not expressed in the amount of tokens on the output but rather in the price (or sqrt price) after the swap.